Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Writers/CloudFormationWriter.cs

```diff
@@ -10,6 +10,7 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using YamlDotNet.RepresentationModel;
```

Comment:
nit: Can you remove this using statement now that it is no longer used.

Reply:
And the one in [JsonWriter.cs](https://github.com/aws/aws-lambda-dotnet/pull/1532/files#diff-743a8b526e00fd576280ee0173731c5f72116c40bc32306a7e6cded1ad540960) too?

Reply:
Done